.github/workflows/check-changes-ownership.yml

```
@@ -0,0 +1,60 @@
+name: 'Check changes ownership'
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+    branches: [ 'main*' ]
```

Q: do you use other branches than `main` exactly? 🤔 
If the answer is no then using limiting it to `main` may be a good idea 🙂 
```suggestion
    branches: [ 'main' ]
```